client/post-editor/editor-ground-control/history-button.jsx

```diff
@@ -0,0 +1,27 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { NESTED_SIDEBAR_REVISIONS } from 'post-editor/editor-sidebar/constants';
+
+const HistoryButton = ( { selectRevision, setNestedSidebar, translate } ) => (
+	<div className="editor-ground-control__history">
+		<button
+			className="editor-ground-control__save button is-link"
+			onClick={ function() {
+				selectRevision( null );
+				setNestedSidebar( NESTED_SIDEBAR_REVISIONS );
+			} }
+		>
+			{ translate( 'History' ) }
```

Comment:
@shaunandrews -- what do you think about replacing the `History` label here with the History Gridicon & revision count that's presently in the [sidebar](https://github.com/Automattic/wp-calypso/blob/ede86661511f81081365cbbb08f9e5f521a610c7/client/post-editor/editor-revisions/index.jsx#L30-L43)?

<img width="120" alt="screen shot 2017-10-04 at 4 18 45 pm" src="https://user-images.githubusercontent.com/1587282/31197674-d7966d54-a91f-11e7-9723-e45139da3ef7.png">